front/lib/api/data_source_view.ts

```diff
@@ -183,7 +184,11 @@ function filterNodesByViewType(
   switch (viewType) {
     case "documents":
       return nodes.filter(
-        (node) => node.children_count > 0 || node.node_type !== "Table"
+        (node) =>
+          node.children_count > 0 ||
+          node.node_type !== "Table" ||
+          // TODO: replace this with either an "all" viewType or a DEFAULT_VIEWTYPE_BY_CONNECTOR_PROVIDER
```

Comment:
```suggestion
          // TODO(nodes-core): replace this with either an "all" viewType or a DEFAULT_VIEWTYPE_BY_CONNECTOR_PROVIDER
```
nit suggestion, so we can easily match the todo along with all the others at the end of the project if needed